### DISCUSSION TITLE
Emphasis in prompts

### DISCUSSION THREAD

**Discussion by Vi3344** [Acly/krita-ai-diffusion#647] [Q&A]
Hi, I'm new and I wanted to know if it's possible to use emphasis in prompts? For example, in comfyUI it's (word:0.5) and the default value is (word:1.1).
Thanks

**Comment by Danamir** [comment]
I works, and you can even use Ctrl+Up/Down to increase/decrease the weight of the selected token.

**Comment by Vi3344** [comment]
Ok, thanks, I don't know why but it doesn't work with me.

**Reply by Danamir** [reply]
Oh, maybe create a bug report about it. Try to add as much information as relevant, like OS and such.
